ccm_web/client/src/models/FeatureUIData.tsx

```diff
@@ -76,5 +77,25 @@ const allFeatures: FeatureUIGroup[] = [
   { id: 'Sections', title: 'Sections', ordinality: 3, features: [mergeSectionCardProps, createSectionsCardProps] }
 ]
 
+const isAuthorizedForRoles = (userRoles: RoleEnum[], requiredRoles: RoleEnum[], featureDescriptionForLogging: string): boolean => {
```

Comment:
I might change this function name to `hasAuthorizedRole`, or something similar. `isAuthorizedForRoles` sounds strange to me, is hard to grok.

Reply:
The naming is kind of consistent  with the other helper functions.  Do you expect them all to be renamed?

Reply:
No, I like the other names, what they are doing is clear to me. This one just isn't as clear to me on a language level. How can one be authorized for roles? The function is checking whether a user has one or more roles that authorize them to access a feature. So `hasAuthorizedRole` or `hasAuthorizedRoles` seems to better encapsulate the logic, in my opinion.

I won't insist on this change, I just thought I'd share my opinion.

Reply:
Ok I can see how that's confusing. I added this function to support course rename since it's the only feature that doesn't fit the pattern so it's kind of a snowflake.  Does `isAuthorizedForRole` work better?

Reply:
Yeah, I realized that's why the function was necessary when I was thinking about suggesting combining it with `isAuthorizedForFeature`. I wonder if there's a way we could make course renaming a feature without giving it all the same properties as the others. Might be more trouble than it's worth.

To be completely honest, I don't think `isAuthorizedForRole` is much of an improvement, but I dunno, maybe I'm being unreasonable 😆